<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="edDataFolder"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnOpenFolder">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data Folder</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="lblTrip">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnLoadModel">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QTableView" name="tblSurveys"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSites"/>
    <addaction name="actionTrip"/>
   </widget>
   <widget class="QMenu" name="menuSync">
    <property name="title">
     <string>Sync</string>
    </property>
    <addaction name="actionTo_Reefscan"/>
    <addaction name="actionFrom_Aquisition_Hardware"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionHardware_folder"/>
    <addaction name="actionServer_Folder"/>
   </widget>
   <widget class="QMenu" name="menuArchives">
    <property name="title">
     <string>Archives</string>
    </property>
    <addaction name="actionShow_Archives"/>
    <addaction name="actionHardware_Archives"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuSync"/>
   <addaction name="menuConfig"/>
   <addaction name="menuArchives"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSites">
   <property name="text">
    <string>Sites</string>
   </property>
  </action>
  <action name="actionTo_Reefscan">
   <property name="text">
    <string>To Reefscan Server</string>
   </property>
  </action>
  <action name="actionShow_Archives">
   <property name="text">
    <string>Local Archives</string>
   </property>
  </action>
  <action name="actionTrip">
   <property name="text">
    <string>Trip</string>
   </property>
  </action>
  <action name="actionFrom_Aquisition_Hardware">
   <property name="text">
    <string>From Aquisition Hardware</string>
   </property>
  </action>
  <action name="actionHardware_Archives">
   <property name="text">
    <string>Hardware Archives</string>
   </property>
  </action>
  <action name="actionHardware_folder">
   <property name="text">
    <string>Hardware folder</string>
   </property>
  </action>
  <action name="actionServer_Folder">
   <property name="text">
    <string>Server Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
